src/main/resources/static/js/results.js

```diff
@@ -0,0 +1,51 @@
+var stompClient = null;
+
+//Requests the CSRF token and makes
+//the connection after receiving a response
+function connect() {
+    jQuery.ajax({
+        url: '/csrf',
+        success: connectStomp
+    });
+}
+
+function connectStomp(csrf) {
+    var headerName = csrf.headerName;
+    var token = csrf.token;
+    
+    var headers = {};
+    headers[headerName] = token;
+    
+    stompClient = Stomp.over(new SockJS('/websocket'));
+    stompClient.connect(headers, function () {
+        stompClient.subscribe('/topic/results', showResults);
+    });
+}
+
+function showResults(results) {
+    var testOutput = JSON.parse(JSON.parse(results.body).testOutput)
+    var status = testOutput.status;
+    var results = testOutput.testResults;
+    
+    $("#results").html("<div>"+ status +"</div>");
```

Comment:
Generating HTML like this causes cross-site scripting vulnerabilities (i.e. what would happen if the value of `status` was `<script>alert(1)</script>` instead of just `FAILED`?), so try to avoid it if possible.

You can do something like this to safely build new elements with jQuery:
```
$("#results").append(
    $("<div></div>")
        .text(status)
);
```